packages/payload/src/bin/generateTypes.ts

```diff
@@ -36,6 +36,52 @@ export async function generateTypes(
     unreachableDefinitions: true,
   })
 
+  const modifiedLines = []
```

Comment:
Since nobody on our team is going to easily be able to maintain this, we should move everything from L39 - L84 to a new function in the utilities folder and add some unit tests too. Otherwise we won't be able to make changes with any confidence or have to do manual testing later.